seirmo/figures.py

```diff
@@ -0,0 +1,99 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import plotly.graph_objects as go
+
+
+class IncidenceNumberPlot(object):
+    """
+
+    Creates incidence number plots for the app
```

Comment:
This figure class could be used also outside of the app that we are writing, so it's better to not reference the app in the docstring. 

Maybe something like:
A figure class that illustrates the incidence number time series as a bar plots over time.